src/GovLst.sol

```diff
@@ -1218,7 +1218,7 @@ abstract contract GovLst is IERC20, IERC20Metadata, IERC20Permit, Ownable, Multi
     uint256 _newShares = _calcSharesForStake(_amount, _totals);
```

Comment:
*arbitrary comment location*

One thing I noticed while reviewing is that we have a whole bunch of downcasts to uint96 before passing to Staker operations, but these are not necessary because Staker takes a uint256. Maybe this is a holdover from porting from UniStaker? I'm actually not sure what would happen if one of those unneeded downcasts were to overflow. Perhaps we should remove them? Or check with OBS to see if its considered safe. Semantically it would probably be better to remove them either way.

Reply:
Handled by https://github.com/withtally/stGOV/pull/66/commits/5cd21c99599aeae2bfb5f7904179d91b75a02957